e2e-tests/cypress/support/check-href.js

```diff
@@ -0,0 +1,3 @@
+export default (selector, expectedHref) => {
+  selector.should('have.attr', 'href', expectedHref);
```

Comment:
Nice! Can we add `text` to this as well?

So `checkHref` will also call `checkText` 😎